tests/acceptance/features/coreApiWebdavOperations/refuseAccess.feature

```diff
@@ -27,7 +27,7 @@ Feature: refuse access
     Given using <dav-path-version> DAV path
     And user "Alice" has been created with default attributes and without skeleton files
     And user "Alice" has uploaded file "filesForUpload/textfile.txt" to "textfile0.txt"
-    And user "Alice" has been disabled
+    And the user "Admin" has disabled user "Alice" using the Graph API
```

Comment:
Thi should be fine I don't think we need to mention the `using the Graph API` here. The stap should use the graph API internally? I don't think there's any other api that can be used to `enable/disable` user apart from graph?
```suggestion
    And the user "Admin" has disabled user "Alice"
```

Reply:
There are ten other steps where `using the Graph API` is used. Should I refactor them in this PR along with the step definition code?

Reply:
No, do that in the next PR.

Reply:
Yeas, let's merge this